<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameWAP</class>
 <widget class="QFrame" name="FrameWAP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>506</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>5006</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Panel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxWAP">
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: rgb(196, 160, 0);
    margin-top: 29px;
    font-size: 16px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 150px;
    color: rgb(255, 255, 0);
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
}</string>
       </property>
       <property name="title">
        <string>Weapon Assign</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Weapon</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxWAPWeapon">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>40 mm</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Mode</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxWAPMode">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>-</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Direct</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>51</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxTrackEng">
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: rgb(196, 160, 0);
    margin-top: 29px;
    font-size: 16px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 150px;
    color: rgb(255, 255, 0);
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
}</string>
       </property>
       <property name="title">
        <string>Track Engagement</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>TN</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBoxTrackEngTN">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>32</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>18</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Weapon</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxTrackEngWeapon">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>40 mm</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonTrackEngAssign">
            <property name="text">
             <string>Assign/Break</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxCorr">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: rgb(196, 160, 0);
    margin-top: 29px;
    font-size: 16px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 150px;
    color: rgb(255, 255, 0);
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
}</string>
     </property>
     <property name="title">
      <string>Correction</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidgetCorrection">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: yellow;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Weapon</string>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Az Corr</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>El Corr</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>40 mm</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>-0.2</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>0.4</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonCorrectionApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxEng">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: rgb(196, 160, 0);
    margin-top: 29px;
    font-size: 16px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 150px;
    color: rgb(255, 255, 0);
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
}</string>
     </property>
     <property name="title">
      <string>Engagement Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidgetEngData">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: yellow;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Weapon</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Engage</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Az</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Elv</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>40 mm</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>Not engage</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>-</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string>-</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
